app/view_models/profile_index.rb

```diff
@@ -16,13 +16,21 @@ def personal_key_partial
   end
 
   def password_reset_partial
-    if current_user.password_reset_profile.present?
+    if current_user.decorate.password_reset_profile.present?
```

Comment:
If it's ok with you @jessieay I'm going to merge this as-is and we can revisit the API for that class in a future PR?